roles/network_plugin/flannel/templates/docker-network-options-systemd

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment="DOCKER_NETWORK_OPTIONS=--bip={{ flannel_subnet }} --mtu={{ flannel_mtu }}"
```

Comment:
Did I get the change right: that DOCKER_NETWORK_OPTIONS is a uniform option for all OS types, including CoreOS, and it replaces DOCKER_OPTS and OPTIONS variations, which are OS type specific?